packages/modal-ui/src/lib/components/styles.css

```diff
@@ -236,7 +236,7 @@
 }
 
 .nws-modal-wrapper .nws-modal .wallet-options-wrapper .options-list .single-wallet.sidebar:hover {
-    background-color: var(--wallet-selector-wallet-option-bg-hover, var(--wallet-option-bg-hover));
+    background-color: var(--selected-wallet-bg, var(--wallet-option-bg-hover));
```

Comment:
Why are we changing the `hover` state background color here?

This looks confusing between `selected/active` and `hover` states and is not really nice for the eyes.

In my opinion, the way we have it right now looks better @amirsaran3  @DamirSQA what do you think?
![selected-active](https://user-images.githubusercontent.com/95851345/196782616-778dd03c-061b-4918-96d4-c7c76e1bec45.png)

Reply:
This has not been resolved yet, please don't mark it as resolved until you commit your changes.